### PR TITLE
add: ChessAnalysisPipeline recipe

### PR DESCRIPTION
**Pull request by keara-soloway**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by keara-soloway** [comment]
I confirm that I am willing to be listed as a maintainer for this recipe.

**Comment by keara-soloway** [comment]
@conda-forge-admin, please ping team

File: recipes/ChessAnalysisPipeline/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{% set name = "ChessAnalysisPipeline" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
```
Comment (ocefpaf) [inline recipes/ChessAnalysisPipeline/meta.yaml:6]:
```suggestion
  name: {{ name|lower }}
  version: {{ version }}
```

File: recipes/ChessAnalysisPipeline/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (ocefpaf) [inline recipes/ChessAnalysisPipeline/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/ChessAnalysisPipeline/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  sha256: 8edc4e86bf0760bc3d7ae4d46efd39ca25c2295baa3cc34e00d78d29329677b2
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (ocefpaf) [inline recipes/ChessAnalysisPipeline/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

File: recipes/ChessAnalysisPipeline/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 8edc4e86bf0760bc3d7ae4d46efd39ca25c2295baa3cc34e00d78d29329677b2
+
+build:
+  number: 0
+  noarch: python
```
Comment (ocefpaf) [inline recipes/ChessAnalysisPipeline/meta.yaml:14]: It ha scripts and cannot be noarch.

```suggestion
  skip: true  # [py<38]
```

File: recipes/ChessAnalysisPipeline/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - CHAP
+    - MLaaS
+  commands:
+    - CHAP --help
+
+about:
+  home: "https://github.com/CHESSComputing/ChessAnalysisPipeline"
```
Comment (ocefpaf) [inline recipes/ChessAnalysisPipeline/meta.yaml:33]:
```suggestion
  home: https://github.com/CHESSComputing/ChessAnalysisPipeline
```

File: recipes/ChessAnalysisPipeline/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "CHESS analysis pipeline framework"
```
Comment (ocefpaf) [inline recipes/ChessAnalysisPipeline/meta.yaml:37]:
```suggestion
  summary: CHESS analysis pipeline framework
```

**Review by ocefpaf — CHANGES_REQUESTED** [review]
Try to use grayskull to create your recipe. Most of the changes here are from an auto-generated recipe with  `grayskull pypi --strict-conda-forge ChessAnalysisPipeline`

**Review by ocefpaf — APPROVED** [review]

**Pull request merged**